client/cypress/integration/app.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { AppPage } from '../support/app.po';
+
+const page = new AppPage();
+
+describe('App', () => {
+  beforeEach(() => page.navigateTo());
+
+  it('Should have the correct title', () => {
+    page.getAppTitle().should('contain', 'CSCI 3601 Iteration Template');
+  });
+
+  it('The sidenav should open, navigate to "Users" and back to "Home"', () => {
+    // Before clicking on the button, the sidenav should be hidden
+    page.getSidenav()
+      .should('be.hidden')
+      .and('not.be.visible');
```

Comment:
Is there a semantic difference between `be.hidden` and `not.be.visible`?

Reply:
We had a whole discussion about that above. It does seem like there is a difference: https://stackoverflow.com/questions/17425543/difference-between-hidden-and-notvisible-in-jquery/17426800

Reply:
Neat, thank you!

In that case, would it be all right to only include one of these two assertions? (Just to avoid redundancy.)